gulp/release-tasks/files.js

```diff
@@ -19,6 +20,26 @@ module.exports = function (gulp, plugins, config, es, webpack, path) {
       cdnReplacement: '="$1assets/includes-cdn/',
       localReplacement: '="$1assets/includes-local/',
     };
+    let envFolders = config.release.staticEnv.env;
+
+    //create environment specific files
+    let envStream = lazypipe()
+      .pipe(function () {
+        return plugins.replace(new RegExp('"/assets', 'g'), `"https://${envFolders.prod}/assets`);
+        })
+      .pipe(gulp.dest, `${config.basepath.static}${config.release.staticEnv.envPath}/${envFolders.prod}/assets/${config.versionName}/latest`)
+      .pipe(function () {
+        return plugins.replace(new RegExp(`${envFolders.prod}`, 'g'), `${envFolders.beta}`);
```

Comment:
Why is every environment being rewritten with the next one down? And won't they stack?

Reply:
The way this gulp task works is, 
- takes file content
- replace with the given string
- copy file to dest folder
- pass it on to next task

In this task we are creating 4 environment dependent files to store under [static CDN repo](https://servicesmadesimpler.govnet.qld.gov.au/bitbucket/projects/CDN/repos/static-qld_swe-v3_assets/browse/_env) 
- replace with prod url and copy that file onto prod env folder
- replace with beta url and copy that file onto beta env folder
- replace with test url and copy that file onto test env folder
- replace with dev url and copy that file onto dev env folder

Reply:
That makes sense.